### DISCUSSION TITLE
Issues with Etsy V3 API and Listings Erroring in Both Etsy Seller Apps

### DISCUSSION THREAD

**Discussion by BlaisePx** [etsy/open-api#998] [Q&A]
I'm hoping someone here can help me with a persistent issue I'm facing on my Etsy store.

Recently, we brought our store back online after a long hiatus, and since then, we've been facing issues with a majority of our listings (27 out of 34, to be exact). These listings are throwing error messages in both Etsy Seller Apps. In one app, the message reads "there is a problem with this listing", and the other displays a red "Not Found" banner. What's more, these listings aren't showing up in search results, even when searching by the **exact title** and sorting by **most recent**. They do appear correctly in the shop view though.

![Screenshot_20230426-103451](https://github.com/etsy/open-api/assets/130322548/55ef27b9-163f-4d4f-8125-cd457c74670a)
![Screenshot_20230426-104621](https://github.com/etsy/open-api/assets/130322548/7dfcb09a-9c04-45cb-a7f5-1c71f514d1ae)
![Screenshot_20230426-110624](https://github.com/etsy/open-api/assets/130322548/64bfc13e-157e-4988-b046-5d40c3e75ab2)

We've used the V3 API to upload all listings. The first couple dozen were uploaded in one batch, and the rest were added one at a time. Each listing has similar tags and metadata, all fields are filled out thoroughly, and all images and video are uploaded successfully.

Also our store was inactive for about nine years and when we first logged back in it was in suspended mode due to the need for updates. I'm wondering if this long hiatus might be a factor in the current issue.

I've tried reaching out to Etsy's customer support, but haven't found a resolution yet. The issue has been ongoing since April 25th, and it's been frustrating, to say the least. As this might be a backend or API problem, I thought it best to seek help from this community. This situation is greatly affecting our business, and we're eager to find a solution.

**Comment by Gareth064** [comment]
Hi @BlaisePx 

Sorry to hear you're having issues with your listings.

Honestly, this looks more like a platform issue that you need help with rather than the API itself. 

You could try your luck though and email this issue over the developer@etsy.com to see if they can help, or at best they can direct you to the correct team for help.
